Working Mode.md

```diff
@@ -32,6 +32,14 @@ The editor has the following responsibilities, all of which can be delegated to
    * Only add features that have sufficient implementer interest.
    * Align existing features with implementations (e.g., via removal if implementer interest waned).
 
+## Meetings
+
+The WHATWG doesn't generally run any regular meetings. However, as circumstances arise, meetings can be helpful to resolve difficult issues, or to liase with other groups.
+
+Requests for meetings are to be filed as issues against the relevant workstream. To allow maximum participation in a meeting, meetings should be announced early and provide an option for remote participation if possible. Transcripts, minutes, or recordings of a meeting should be made available shortly after the meeting concludes (linking from the original meeting request issue is a good way to do this).
```

Comment:
```suggestion
Announcements or requests for meetings are to be filed as issues against the relevant Workstream. To allow maximum participation in a meeting, meetings should be announced early and provide an option for remote participation if possible. Transcripts, minutes, or recordings of a meeting should be made available shortly after the meeting concludes (linking from the original meeting request issue is a good way to do this).
```

Reply:
relevant workstream -> relevant Living Standard repository?

Reply:
Added "Announcements or..."

For "workstream", since "Living Standard repository" is more specific, I took that suggestion.